test/acceptance/environments/local.yml

```diff
@@ -101,3 +101,13 @@ httpMockUrl: http://localhost:9000
 elasticsearch:
   addresses:
     - http://localhost:9200
+
+# s3
+s3-autoclean: true
+minio: true
+bucket: golium-demo
+# Below config is only required by minio server
+minio-endpoint: http://localhost:9000
+minio-aws-access-key-id: minioadmin
+minio-aws-secret-access-key: minioadmin
+minio-aws-region: eu-west-1
```

Comment:
May use `camelCase` instead of `kebab-case` for configuration?

Also, ain't better to prefix all s3 related steps with `s3` insead of `minio`?

Reply:
Any case other than minio will use env vars: AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_REGION. I'd leave minio prefix.

Reply:
As I said in previous command, think that an user may want to override also AWS environment configuration.